src/main/java/com/programmers/kwonjoosung/springbootbasicjoosung/controller/voucher/request/VoucherDto.java

```diff
@@ -0,0 +1,33 @@
+package com.programmers.kwonjoosung.springbootbasicjoosung.controller.voucher.request;
+
+import com.programmers.kwonjoosung.springbootbasicjoosung.model.voucher.Voucher;
+
+public class VoucherDto {
+    private String voucherId;
+    private String voucherType;
+    private long discount;
+
+    public VoucherDto(String voucherId, String voucherType, long discount) {
+        this.voucherId = voucherId;
+        this.voucherType = voucherType;
+        this.discount = discount;
+    }
+    public VoucherDto(Voucher voucher) {
+        this.voucherId = voucher.getVoucherId().toString();
+        this.voucherType = voucher.getVoucherType().toString();
+        this.discount = voucher.getDiscount();
```

Comment:
생성자 this를 사용하여 초기화하면 더 깔끔하게 처리할 수 있을 것 같습니다.
```suggestion
        this(voucher.getVoucherId().toString(), voucher.getVoucherType().toString(), voucher.getDiscount());
```